clouditor-engine-core/src/main/java/io/clouditor/auth/User.java

```diff
@@ -145,4 +149,56 @@ public void setFullName(String fullName) {
   public String getEmail() {
     return email;
   }
+
+  @Override
+  public String toString() {
+    return "User{"
+        + "username='"
+        + username
+        + '\''
+        + ", password='"
+        + password
+        + '\''
+        + ", fullName='"
+        + fullName
+        + '\''
+        + ", email='"
+        + email
+        + '\''
+        + ", shadow="
+        + shadow
+        + ", roles="
+        + roles
+        + '}';
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
+
+    if (o == null || getClass() != o.getClass()) return false;
+
+    User user = (User) o;
+
+    return new EqualsBuilder()
```

Comment:
There was a comment in the original code, that comparison should not include password. Not sure why but there was probably a good reason for it